pkg/ddevapp/drupal/drupal7/settings.ddev.php

```diff
@@ -29,6 +29,13 @@
 
 $drupal_hash_salt = '{{ $config.HashSalt }}';
 
+// This will prevent Drupal from setting read-only permissions on sites/default.
+$conf['skip_permissions_hardening'] = TRUE;
```

Comment:
Last I checked, Drupal 7 didn't have this feature. It would have saved lots of pain if it had! It's important! 

Drupal's ridiculous attempt at security by making sites/default and sites/default/settings.php non-writable during an install has annoyed me for a really, really long time :)